src/sentry/static/sentry/app/views/settings/components/forms/inputField.jsx

```diff
@@ -22,13 +22,12 @@ export default class InputField extends React.Component {
   };
 
   render() {
-    let {className, field, ...otherProps} = this.props;
+    let {className, field} = this.props;
 
     return (
       <FormField className={className} {...this.props}>
         {({children, ...formFieldProps}) => {
           return field({
-            ...otherProps,
```

Comment:
We want `<FormField>` to be able to control what props its rendered child should be able to receive, passing `otherProps` defeats the purpose of it. An example is that we render `<InputField defaultValue="..."/>`, FormField ingests `defaultValue`, but always controls the input field using `value`.

Reply:
Makes sense, do you know if there are any other places that are gonna break as a result of changing this?

Reply:
It shouldn't have an effect in other places because the control (`field` in the above code) is already being passed everything besides these props (which `FormField` destructures):

```
      hideErrorMessage,
      flexibleControlStateSize,
      saveOnBlur,
      saveMessage,
      saveMessageAlertType,
```

And these don't get used by the form control. `defaultValue` is the first case where we want to make sure it doesn't get passed to the control, otherwise it'll throw React warnings about receiving both a `defaultValue` and `value`.